src/locales/en.json

```diff
@@ -577,7 +581,7 @@
   "tx_modal_status_complete_text_collect_portfolio": "your portfolio",
   "tx_modal_status_complete_text_collect_without_wallet": "You will receive an email from us soon. Please follow the instructions in the email to claim your Writing NFT.",
   "tx_modal_status_complete_text_transfer": "Your Writing NFT is successfully transferred!",
-  "tx_modal_status_complete_title": "Payment is successful!",
+  "tx_modal_status_complete_title": "Transaction is complete!",

```

Comment:
I think we changed this string to avoid the word transaction for the whole collect ux

Reply:
This is the fallback message, collect modal should be using `collect_modal_complete_title`